src/main/java/com/exasol/adapter/document/mapping/IterationIndexColumnMapping.java

```diff
@@ -57,6 +54,95 @@ public void accept(final ColumnMappingVisitor visitor) {
+        private DocumentPathExpression tablesPath;
+
+        @Override
+        protected B fillValuesFrom(final C instance) {
+            super.fillValuesFrom(instance);
+            IterationIndexColumnMapping.IterationIndexColumnMappingBuilder.fillValuesFromInstanceIntoBuilder(instance,
+                    this);
+            return self();
+        }
+
+        private static void fillValuesFromInstanceIntoBuilder(final IterationIndexColumnMapping instance,
+                final IterationIndexColumnMapping.IterationIndexColumnMappingBuilder<?, ?> b) {
+            b.tablesPath(instance.tablesPath);
+        }
+
+        @Override
+        protected abstract B self();
+
+        @Override
+        public abstract C build();
+
+        /**
+         * @param tablesPath table path
+         * @return {@code this}.
+         */
+        public B tablesPath(final DocumentPathExpression tablesPath) {
+            this.tablesPath = tablesPath;
+            return self();
+        }
+
+        @Override
+        public String toString() {
+            return "IterationIndexColumnMapping.IterationIndexColumnMappingBuilder(super=" + super.toString()
+                    + ", tablesPath=" + this.tablesPath + ")";
+        }
+    }
+
+    private static final class IterationIndexColumnMappingBuilderImpl extends
+            IterationIndexColumnMapping.IterationIndexColumnMappingBuilder<IterationIndexColumnMapping, IterationIndexColumnMapping.IterationIndexColumnMappingBuilderImpl> {
+
+        private IterationIndexColumnMappingBuilderImpl() {
+        }
+
+        @Override
+        protected IterationIndexColumnMapping.IterationIndexColumnMappingBuilderImpl self() {
+            return this;
+        }
+
+        /**
+         * Build a new instance.
+         * 
+         * @return new instance
+         */
+        @Override
+        public IterationIndexColumnMapping build() {
+            return new IterationIndexColumnMapping(this);
+        }
+    }
+
+    /**
+     * Creates a new instance from a builder.
+     * 
+     * @param builder builder
+     */
+    protected IterationIndexColumnMapping(
+            final IterationIndexColumnMapping.IterationIndexColumnMappingBuilder<?, ?> builder) {
+        super(builder);
+        this.tablesPath = builder.tablesPath;
+    }
+
+    /**
+     * Create a new builder for {@link IterationIndexColumnMapping}.
+     * 
+     * @return a new builder
+     */
+    @SuppressWarnings("java:S1452") // Generic wildcard type is ok here
+    public static IterationIndexColumnMapping.IterationIndexColumnMappingBuilder<?, ?> builder() {
+        return new IterationIndexColumnMapping.IterationIndexColumnMappingBuilderImpl();
+    }
+
+    private IterationIndexColumnMapping.IterationIndexColumnMappingBuilder<?, ?> toBuilder() {
+        return new IterationIndexColumnMapping.IterationIndexColumnMappingBuilderImpl().fillValuesFrom(this);
```

Comment:
```suggestion
        return builder().fillValuesFrom(this);
```

Reply:
Not possible, because `fileValuesFrom()` is only available in the concrete impl class